konfera/models/ticket.py

```diff
@@ -30,3 +32,14 @@ def __str__(self):
             first_name=self.first_name,
             last_name=self.last_name
         ).strip()
+
+    def save(self, *args, **kwargs):
+        if not hasattr(self, 'order'):
+            discount = 0
+            if self.discount_code:
+                discount = self.type.price * self.discount_code.discount/100
+            order = Order(price=self.type.price, discount=discount, status='awaiting_payment',
```

Comment:
Using string instead of constant is bad practice. Please import from order AWAITING and use constant for status.